lightbox.js

```diff
@@ -140,8 +146,17 @@ function bind() {
   });
 }
 
-function init(context) {
-  $context = $(context || document.body);
+function init(options) {
```

Comment:
Good point, I should probably bump to 2.0.0. Changing it to options makes it more flexible in case we want to add more options in the future.